mifospay/src/main/java/org/mifos/mobilewallet/mifospay/faq/ui/FAQActivity.kt

```diff
@@ -18,97 +19,55 @@ import javax.inject.Inject
  * @since 11/July/2018
  */
 class FAQActivity : BaseActivity(), FAQView {
-    @JvmField
+
     @Inject
-    var mPresenter: FAQPresenter? = null
+    lateinit var mPresenter: FAQPresenter
```

Comment:
We don't need presenter, delete it and create FaqViewModel as suggested in another comment.